### DISCUSSION TITLE
Account Suspension

### DISCUSSION THREAD

**Discussion by Moksh-Bhardwaj** [community/community#35573] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi, my GitHub account has been suspended by GitHub without any warnings & notice.
My GitHub account was 1 year old & i was working on it from 1 year.
I added all the details in my GitHub account for security reasons like phone number 2 factor authentication & all.
But still GitHub suspended my account suddenly can anyone help me to reactivate my account.

**Comment by radar-blip** [comment, marked as answer]
For account related issues, try here: [Support page](https://support.github.com/contact/account)

**Comment by JasonLinnnnnn** [comment]
Hey there, do you get it back?

**Reply by Moksh-Bhardwaj** [reply]
No they denied to return it.

**Comment by klaudiusmiskinis** [comment]
Did you make more then 1 account? [Github Policy](https://docs.github.com/en/site-policy/github-terms/github-terms-of-service#3-account-requirements)

People spamming in your repos could be a cause too.

People which had this exact problem contacted with support and got it solved by that.

**Reply by JasonLinnnnnn** [reply]
never suffer the cases you metioned...
This's just creating a new accout via another email,  to help to find others who have solved the same issue...
whatever, thanks for your reply!

**Comment by queenofcorgis** [comment]
Hi there - the Community is unable to assist in account-related matters like these. We'll be locking this post as queries related to flagged accounts should go through our [Support team](https://support.github.com/).
